pilot/test/integration/cloudfoundry/copilot_test.go

```diff
@@ -43,16 +43,48 @@ import (
 )
 
 const (
-	appPort           = 9000
+	appPort           = 61005
 	pilotPort         = 5555
 	copilotAddr       = "127.0.0.1:5000"
 	pilotAddr         = "127.0.0.1:5555"
 	registrationRoute = "http://127.0.0.1:5555/v1/registration"
 	listenersRoute    = "http://127.0.0.1:5555/v1/listeners/x/router~x~x~x"
-	appEnvoyListener  = "http://127.0.0.1:6666"
+	appEnvoyListener  = "http://127.0.0.1:8443"
 )
 
-func TestEdgeRouterWithMockCopilot(t *testing.T) {
+var routeRule = `
+apiVersion: config.istio.io/v1alpha2
+kind: V1alpha2RouteRule
+metadata:
+  name: ultimate-route-rule
+spec:
+  hosts:
+  - abc.com
+  gateways:
+  - ultimate-gateway
+  http:
+  - route:
+    - destination:
+        port:
+          number: 6666
+        name: abc.com
+`
+
+var gatewayConfig = `
+apiVersion: config.istio.io/v1alpha2
+kind: Gateway
+metadata:
+  name: ultimate-gateway
+spec:
+  servers:
+  - port:
+      number: 8443
+      protocol: http
+    hosts:
+    - abc.com
+`
+
```

Comment:
In general I don't mind one way or another. Its just that I am trying to keep the code consistent. I have no strong preference here. Will let others decide.